Mathlib/Topology/Bases.lean

```diff
@@ -354,6 +354,15 @@ theorem _root_.Topology.IsQuotientMap.separableSpace [SeparableSpace α] [Topolo
     {f : α → β} (hf : IsQuotientMap f) : SeparableSpace β :=
   hf.surjective.denseRange.separableSpace hf.continuous
 
+theorem _root_.IsOpenMap.separableSpace_of_injective [TopologicalSpace β] [SeparableSpace β]
+    {f : α → β} (h : IsOpenMap f) (h' : Function.Injective f) : SeparableSpace α :=
```

Comment:
I guess in a follow-up this could be generalized to `f` having countable fibers.
But this version is also useful, because in applications I guess `h'` might be floating around.